guide/index.html

```diff
@@ -19,58 +19,68 @@
-    <script src="js/navbar.js"></script>
   </head>
 
   <body id="home">
 
     <header>
       <!-- Global Omnibar -->
       <div class="omnibar">
-        <nav class="navbar navbar-expand-lg">
+        <nav class="navbar">
           <a class="navbar-brand" href="https://stjude.cloud">
             <span class="logo">
-              <img src="images/stjude-logo-child.png" alt="St. Jude Cloud" title="St. Jude Cloud" />
+              <img src="../dist/images/stjude-logo-child.png" alt="St. Jude Cloud" title="St. Jude Cloud" />
             </span>
           </a>
 
           <span class="sjc-title">
             <a href="https://stjude.cloud">St. Jude Cloud</a>
           </span>
           <span class="site-title mr-auto">
-            <span>Designer and Developer Guide</span>
+            <span>App Title</span>
           </span>
 
           <ul class="nav justify-content-center global-icons">
-            <li class="nav-item active" id="icon-platform" title="About St. Jude Cloud">
-              <a class="nav-link" href="https://stjude.github.io/sjcloud-docs/index.html">
-                <i class="fa fa-info-circle"></i>
-              </a>
-            </li>
+            
             <li class="nav-item nav-item-apps dropdown">
               <a class="nav-link dropdown-toggle" href="/" id="nav-item-icon-apps" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="St. Jude Cloud Apps">
-                <i class="fa fa-th"></i>
+                <i class="fas fa-bars"></i>
               </a>
-              <div class="dropdown-menu" aria-labelledby="nav-item-icon-apps">
-                <a class="dropdown-item" href="http://cstn-ui.azurewebsites.net">CSTN</a>
-                <a class="dropdown-item" href="http://nbtp-dev.azurewebsites.net">NBTP</a>
-                <a class="dropdown-item" href="https://pecan.stjude.cloud">Pecan</a>
-                <a class="dropdown-item" href="http://sicklecelldev.azurewebsites.net">Sickle Cell</a>
-              </div>
+              <div class="dropdown-menu dropdown-menu-right dropdown-menu-main" aria-labelledby="nav-item-icon-waffle" id="dropdown-main">
+                <div class="d-flex flex-column flex-lg-row">
+                  <div class="nav-flex-fill">
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/data"><img alt="data-icon" class="icon-nav" src="../dist/images/icon-nav-data.png">Data Browser</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/tools"><img alt="tools-icon" class="icon-nav" src="../dist/images/icon-nav-tools.png">Analysis Tools</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/visualizations"><img alt="visualizations-icon" class="icon-nav" src="../dist/images/icon-nav-visualization.png">Curated Visualizations</a>
+                  </div>
+                  <div class="nav-flex-fill">

```

Comment:
```suggestion
                  <div class="nav-flex-fill">
                   <p class="dropdown-column-header">Portals</p>
                    <a class="dropdown-item" href="https://pecan.stjude.cloud">PeCan</a>
                    <a class="dropdown-item" href="http://sickle-cell.stjude.cloud/">Sickle Cell Disease</a>
                    <a class="dropdown-item" href="https://proteinpaint.stjude.org/survivorship/">Survivorship</a>
                  </div>
```

Reply:
So I really don't like the word "Portals", but my feeling is it's better to use it here as an intermediate solution and then transition to using "Apps" once we make the full mental model switch. That way, we won't confuse future users when "Survivorship" and "Sickle Cell Disease" are no longer considered Apps at the next transition.

For PeCan, I think it's sufficient to just say the name without having the parenthesis (makes it verbose and sort of choppy IMO).